src/components/MMSearch/MMSearch.component.tsx

```diff
@@ -163,6 +165,13 @@ export const MMSearch = (props: MMSearchProps) => {
                 label={label}
                 iconName={leadingIcon}
                 onClose={() => {
+                    if (onClearInput) {
+                        onClearInput();
+                    }
+                    if (inputRef.current) {
```

Comment:
if there need to check whether inputRef is null or not ?

Reply:
yes, inputRef may be undefined and throws error if not checked